config/swagger.yaml

```diff
@@ -175,7 +122,7 @@ paths:
       parameters:
         - in: body
           name: body
-          description: Put a new block to node
+          description: A new block
```

Comment:
I see some have description in single quotes, some without.

Reply:
I left the quotation as it was originally. Swagger does not demand one or the other.

Reply:
I think quotation is required only when there are special symbols like `'`